README.md

```diff
@@ -36,6 +36,18 @@ This project requires Docker and access to DockerHub.
 The OpenAPI Specification is compatible with [Conjur OSS v1.9+](https://github.com/cyberark/conjur).  
 Clients are generated using [OpenAPI Generator v4.3.1](https://github.com/OpenAPITools/openapi-generator/tree/v4.3.1).
 
+### Use With Dynamic Access Provider (DAP)
+
+Conjur's endpoints are a strict subset of DAP's endpoints, for that reason this API specification
```

Comment:
is this true? for example, when you curl `{conjur url}/` - you get the HTTP info page for Conjur OSS. what does the appliance return when you query that endpoint? are there others that work one way for OSS but work differently for DAP? worth verifying / being clear about it in this section

Reply:
You are correct, I updated the wording to be more clear that the endpoints included in the spec are a subset of DAP's endpoints.